analyses/fusion_filtering/results/pbta-fusion-recurrent-fusion-byhistology.tsv

```diff
@@ -1,6 +1,5 @@
 FusionName	broad_histology	count
-KIAA1549--BRAF	Low-grade astrocytic tumor	109
+KIAA1549--BRAF	Low-grade astrocytic tumor	114
```

Comment:
I'm assuming these results changes are expected, but just going to bring them to attention so we can check.

Reply:
yup, these are expected since some samples were updated to Low-grade. More info in Jo Lynne's comment here https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/865#issuecomment-740314937